### PR TITLE
added function for finding top 3 cities and updated data

### PR DESCRIPTION
**Pull request by crgjr66**
- form is now populated with top 3 cities based on search criteria for a single topic

### CODE REVIEW

File: app/javascript/components/pages/SearchNew.js [hunk 1]
```diff
@@ -41,14 +44,61 @@ class SearchNew extends React.Component {
...
+      let thirdCity = ""
+      let topValue = 0.0
+      let secondValue = 0.0
+      let thirdValue = 0.0
+      console.log(this.searchKey)
+      console.log(this.searchValue)
+      cityData.filter(value => {
+        for (var i in value) {
+          if (i == this.searchKey) {
+            if (value[i] > this.searchValue) {
```
Comment (zan-rosenthal) [inline app/javascript/components/pages/SearchNew.js]: It's a little difficult to follow what's happening here. I think renaming some of these variables to be more descriptive about what they are might be an easy way to improve readability. Maybe renaming `value` to `city`  and `i` to `key` or `field`, etc. 

I think also, pulling this filtering function out into a separate function that takes the `searchValue` and `searchKey` and returns `topCity`, `secondCity`, and `thirdCity`. 

Lastly, if you have time to write a test for that function, I think this is good place for testing. It's a little difficult to read and there's a lot of variable mutation going on, which can generally be a source of bugs. You could follow up later with this if you don't have time, but if you do think there will be changes made to this logic in the course of the development, having tests around it is going to make those changes easier and faster.

**Review by zan-rosenthal — CHANGES_REQUESTED** [review]
@crgjr66 Can you add `.DS_Store` to the .gitignore and remove the `.DS_Store` files from the diff here?

File: app/javascript/components/pages/SearchNew.test.js [hunk 2]
```diff
@@ -0,0 +1,253 @@
+// Write the test for a function that returns "drink coffee" if you are
```
Comment (sjproctor) [inline app/javascript/components/pages/SearchNew.test.js:1]: I'm guessing this is an accident?

**Review by sjproctor — APPROVED** [review]
You have an extra jest file in here, but everything else looks good.

**Pull request merged**